Improve texts and remove "we" (#3457)
withastro/docs: src/content/docs/en/reference/publish-to-npm.mdx

## Quick Start

> To get started developing your component quickly, you can use a template already setup for you.

Reviewer: ```suggestion
To get started developing your component quickly, you can use a template already setup for you.
```
I do like this better without the we, but as mentioned in another comment, this is not a case of "wrong we" usage.

:::

> To create a new package, you should configure your development environment to use **workspaces** within your project. This will allow you to develop your component alongside a working copy of Astro.

Reviewer: ```suggestion
To create a new package, configure your development environment to use **workspaces** within your project. This will allow you to develop your component alongside a working copy of Astro.
```
We don't use "should" in the docs because it can be confusing advice. It *sounds* like you "must" (in which case, we write "must" or something else similarly absolute), but it doesn't *mean* that you must.

I'm OK making this an absolute statement here as above. Note: the original was actually fine because the "we" being used was "Astro / Docs people". We don't use "we" when writing instructions for someone *else* to follow, because "we" are not there. But, it's perfectly fine to say "we" when you don't include the reader, and are instead saying what we at Astro recommend!

      - ... additional files used by the package
</FileTree>

This example, named `my-project`, creates a project with a single package, named `my-component`, and a `demo/` directory for testing and demonstrating the component.

> The project root’s `package.json` file:

Reviewer: ```suggestion
This is configured in the project root’s `package.json` file:
```
The above lost the helpful sense of configuring this, so I've reverted the text. Approve of the colon!

}
```

> To use `index.js` as an entrypoint with `import` and `export`, use `"type": "module"`.

Reviewer: ```suggestion
Use `"type": "module"` so that your `index.js` can be used as an entrypoint with `import` and `export` .
```
Again, this lost the cause/effect explanation, so I've gone back to the original structure, but made it shorter without the "we".

## Testing your component

> Astro does not currently ship a test runner. _(If you are interested in helping out tackling this, [join us on Discord!](https://astro.build/chat/))_

Reviewer: I see what you did there.... ;)  :partying_face:

> Astro does not currently ship a test runner. _(If you are interested in helping out tackling this, [join us on Discord!](https://astro.build/chat/))_

Reviewer: ```suggestion
Astro does not currently ship a test runner. _(If you are interested in helping out with this, [join us on Discord!](https://astro.build/chat/))_
```
I think this plainer language is more suitable, especially when we consider translating it. But, totally approve that docs is not the right place to say what we think we would like to do!

## Publishing your component

Once you have your package ready, you can publish it to npm using the `npm publish` command. If that fails, make sure that you have logged in via `npm login` and that your `package.json` is correct. If it succeeds, you're done!

> Notice that there was no `build` step for Astro packages. Any file type that Astro supports natively, such as `.astro`, `.ts`, `.jsx`, and `.css`, can be published directly without a build step. 

Reviewer: Nice change!

> Notice that there was no `build` step for Astro packages. Any file type that Astro supports natively, such as `.astro`, `.ts`, `.jsx`, and `.css`,  can be published directly without a build step. 

Author: ```suggestion
Notice that there was no `build` step for Astro packages. Any file type that Astro supports natively, such as `.astro`, `.ts`, `.jsx`, and `.css`, can be published directly without a build step. 
```
